.github/workflows/blueprints.yaml

```diff
@@ -0,0 +1,84 @@
+name: blueprints
+
+on:
+  push:
+    branches:
+      - dev-hilt
+      - hilt
```

Comment:
Nit: we just need `dev-hilt` here

Reply:
My idea is to rename this branch to `hilt` today and make it official, replacing `android-dagger`